### PR TITLE
BUG: UI related changes moved from Versioned to Versioned Admin.

### PR DESCRIPTION
**Pull request by mfendeksilverstripe**
# UI related changes moved from Versioned to Versioned Admin

* no functionality change
* UI related functionality moved to a more appropriate module

## Origin PR
https://github.com/silverstripe/silverstripe-versioned/pull/222

## Related PR
https://github.com/silverstripe/silverstripe-versioned/pull/296

### CODE REVIEW

File: client/src/legacy/VersionedEditForm/VersionedEditForm.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  $('.cms-edit-form .btn-toolbar #Form_ItemEditForm_action_doUnpublish').entwine({
+
+    /**
+     * Function: onclick
+     *
+     * Parameters:
+     *  (Event) e
+     */
+    onclick(e) {
+      const message = 'Are you sure you want to remove your record from the published site?\n\nThis record will still be available in the CMS as draft.';
```
Comment (robbieaverill) [inline client/src/legacy/VersionedEditForm/VersionedEditForm.js:17]: Ideally we can localise this, i.e. `i18n._t('VersionedEditForm.CONFIRM_MESSAGE', 'Are you sure...')`

File: client/src/legacy/VersionedEditForm/VersionedEditForm.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   */
+  $('.cms-edit-form .btn-toolbar #Form_ItemEditForm_action_doArchive').entwine({
+    /**
+     * Function: onclick
+     *
+     * Parameters:
+     *  (Event) e
+     */
+    onclick(e) {
+      const message = 'Warning: This record will be unpublished before being sent to the archive.\n\nAre you sure you want to proceed?';
```
Comment (robbieaverill) [inline client/src/legacy/VersionedEditForm/VersionedEditForm.js:43]: Also here, we should localise this message

**Review by robbieaverill — APPROVED** [review]
I realise this is moving code from versioned to versioned-admin, but I've left some suggestions for you anyway =)

**Comment by mfendeksilverstripe** [comment]
@robbieaverill Added localisation of the messages. Please review.

**Review by robbieaverill — COMMENTED** [review]
Thank you! If you don't mind, one last thing would be to add the definitions for these into the English language dictionary file. We don't have a JavaScript text collector, so we need to compile the JS dictionary by hand =(

**Comment by mfendeksilverstripe** [comment]
@robbieaverill Language files updates, please review.

**Review by robbieaverill — APPROVED** [review]
I appreciate your ongoing commitment to doing the annoying things I've been asking for

**Comment by robbieaverill** [comment]
@Cheddam can you re-trigger the latest 1 branch build here? https://travis-ci.org/github/silverstripe/silverstripe-versioned-admin/builds/724787356 I don't seem to have permission to do it

**Comment by robbieaverill** [comment]
The 6 behat failures exist in the upstream 1 branch too, so not caused by this PR.

**Pull request merged**